chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,786 @@
+such as simulations and bootstrapping.
+Furthermore, all random processes introduce statistical noise
+or uncertainty into estimates of effect sizes.
+Choosing one random sample from all the possibilities produces some probability of
 choosing a group of units that are not, in fact, representative.
-Choosing one final randomization assignment similarly produces some probability of
-creating groups that are not good counterfactuals for each other.
-Power calculation and randomization inference
+Similarly, choosing one random assignment produces some probability of
+creating groups that are not good counterfactuals.
+\textit{Power calculation} and \textit{randomization inference}
 are the main methods by which these probabilities of error are assessed.
-These analytical dimensions are particularly important in the initial phases of development research --
-typically conducted well before any actual field work occurs --
-and often have implications for feasibility, planning, and budgeting.
+These analyses are particularly important in the initial phases of development research --
+typically conducted before any actual field work occurs --
+and have implications for feasibility, planning, and budgeting.
 
-In this chapter, we first cover the necessary practices to ensure that random processes are reproducible.
-We next turn to how to implement sampling and randomized assignment,
-both for simple, uniform probability cases, and more complex designs,
-such as those that require clustering or stratification.
-We include code examples so the guidance is concrete and applicable.
-The last section discusses power calculations and randomization inference,
-and how both are important tools to critically and quantitatively assess different
-sampling and randomization designs and to make optimal choices when planning studies.
+%-----------------------------------------------------------------------------------------------
+\subsection{Randomizing sampling and treatment assignment}
+
+% sampling universe: the master dataset
+\textbf{Sampling} is the process of randomly selecting units-of-observation
+from a list of individuals to create a representative or statistically similar sub-sample.\index{sampling}
+This can, for example, select a subset from all eligible units
+to be included in data collection when the cost of collecting data on everyone is prohibitive.\sidenote{
+	\url{https://dimewiki.worldbank.org/Sample\_Size\_and\_Power\_Calculations}}
+But it can also select a sub-sample of your observations to test a computationally heavy process 
+before running it on the full data.
+\textbf{Randomized treatment assignment} is the process of assigning observations into treatment arms,
+and this is central to experimental research design.
+Most of the code processes used for randomized assignment are the same as those used for sampling,
+since it is also a process of splitting a list of observations into groups.
```

Comment:
```suggestion
since it is also a process of randomly splitting a list of observations into groups.
```

Reply:
```suggestion
since it is also a randomized process of splitting a list of observations into groups.
```
Does my suggestion sound more as a structured process of randomization? Want to avoid that it sounds at all as the millennials' usage of the word "random"